README.md

```diff
@@ -78,22 +78,30 @@ $CONFIG = array (
 );
 ```
 ### Usage with [Keycloak](https://www.keycloak.org/)
-1. Create the NextCloud Client in a Keycloak Realm of your choosing.
-	1. Set `Access type` to `confidential`  
-	Note: The `Client Secret` will be in the newly created `Credentials` tab after saving.
-	2. Add a `Valid Redirect uri` of `https://cloud.example.com/*`
-	3. Add a `Mapper` to the Nextcloud OIDC Client 
-		1. Set `Mapper Type` to `user attribute`
-		2. Set `Name`, `User Attribute`, and `Token Claim Name` to `ownCloudQuota`
-		3. Set `Claim JSON Type` as `string`.
-2. Add a User `Attribute` to necessary user accounts in Keycloak.
-	1. Set `Key` as `ownCloudQuota` and `Value` to your preferred limit.
-3. Necessary `config.php` settings (differing from above)
+1. Create a new Client for Nextcloud in a Keycloak Realm of your choosing.
+    1. Set a `Client ID` and save.
+    2. Set `Access type` to `confidential` 
+	3. Add a `Valid Redirect URI` e.g. `https://cloud.example.com/*`.
+	4. Open the  `Fine Grain OpenID Connect Configuration` dropdown and set `ID Token Signature Algorithm` to e.g. `RS512` and save.
+	
+	Note: Only HS256, HS384, HS512, RS256, RS384, RS512 is supported.
+	
```

Comment:
In my experiments, this wasn't necessary with Nextcloud 18.0.4, Keycloak 9.0.3, and OpenID Connect Login 1.3.1.

Reply:
I haven't used Keycloak, but are there other options here that might not work? AFAIK, the OIDC library we're using doesn't support ECC.

Reply:
For the `Acess/ID Token Signature Algorithms` I see `{ES,HS,PS,RS}{256,384,512}` as possible options in the Keycloak UI. However, the default is empty, and I would assume that means an algorithm supported on both ends is negotiated automatically?
So the default should work and there's a problem only when someone explicitly selects an algorithm in Keycloak that is not supported by the OIDC library?

Reply:
I mean it wont be picked automatically. So the default should not work.

Reply:
> I mean it wont be picked automatically. So the default should not work.

Unless I miss something defaults work for me, as stated initially.

Reply:
> Unless I miss something defaults work for me, as stated initially.

https://github.com/pulsejet/nextcloud-oidc-login/issues/10#issuecomment-588582619

Reply:
Doesn't harm to keep this I guess